.pipelines/cni/cilium/cilium-scale-test.yaml

```diff
@@ -2,6 +2,63 @@ pr: none
 trigger: none
 
 stages:
+  - stage: create_rg_cluster
+    displayName: "Create RG & Cluster"
+    jobs:
+      - job: create_rg_cluster
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+          - task: AzureCLI@2
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -ex
+                az extension add --name aks-preview
+                if [ ${CREATE_RG} ]; then
+                  az group create --name ${RESOURCE_GROUP} --location ${LOCATION}
+                fi
+                subscriptionId="$(az account list --query "[?isDefault].id" --output tsv)"
+                make -C ./hack/aks AZCLI=az ${CLUSTER_TYPE} CLUSTER=${CLUSTER} GROUP=${RESOURCE_GROUP} REGION=${LOCATION} NODE_COUNT=5 VM_SIZE=${VMSIZE} SUB=${subscriptionId}
```

Comment:
What functionality are you looking for?

The cluster creation in `hack/aks` creates a resource group every time.